ui/src/app/base/components/DeprecationNotices/DeprecationNotices.js

```diff
@@ -0,0 +1,42 @@
+import { Notification } from "@canonical/react-components";
+import { useDispatch, useSelector } from "react-redux";
+import React, { useEffect } from "react";
+
+import { general as generalActions } from "app/base/actions";
+import { general as generalSelectors } from "app/base/selectors";
+
+const DeprecationList = () => {
+  const dispatch = useDispatch();
+  const deprecationNotices = useSelector(
+    generalSelectors.deprecationNotices.filterByMinorVersion
+  );
+
+  useEffect(() => {
+    dispatch(generalActions.fetchVersion());
+    dispatch(generalActions.fetchDeprecationNotices());
+  }, [dispatch]);
+
+  return (
+    <>
+      {deprecationNotices.map((notice) => (
+        <Notification
+          data-test={`deprecation-${notice.id}`}
+          key={notice.id}
+          type="caution"
+        >
+          {notice.description}
+          <br />
```

Comment:
Is this a sin?

Reply:
@lyubomir-popov -sanctioned!
https://github.com/canonical-web-and-design/MAAS-design/issues/897#issuecomment-616438523